2023-08-10 13:14:10,832 ERROR: Exception on /jobs [GET] [in C:\Users\jai_s\Desktop\2205Flask\2205Flask\venv\lib\site-packages\flask\app.py:1414]
Traceback (most recent call last):
  File "C:\Users\jai_s\Desktop\2205Flask\2205Flask\venv\lib\site-packages\flask\app.py", line 2190, in wsgi_app
    response = self.full_dispatch_request()
  File "C:\Users\jai_s\Desktop\2205Flask\2205Flask\venv\lib\site-packages\flask\app.py", line 1486, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "C:\Users\jai_s\Desktop\2205Flask\2205Flask\venv\lib\site-packages\flask\app.py", line 1484, in full_dispatch_request
    rv = self.dispatch_request()
  File "C:\Users\jai_s\Desktop\2205Flask\2205Flask\venv\lib\site-packages\flask\app.py", line 1469, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**view_args)
  File "C:\Users\jai_s\Desktop\2205Flask\2205Flask\app\main\routes.py", line 149, in jobs
    return render_template('jobs.html', title='Jobs', job_listings=job_listings)
  File "C:\Users\jai_s\Desktop\2205Flask\2205Flask\venv\lib\site-packages\flask\templating.py", line 151, in render_template
    return _render(app, template, context)
  File "C:\Users\jai_s\Desktop\2205Flask\2205Flask\venv\lib\site-packages\flask\templating.py", line 132, in _render
    rv = template.render(context)
  File "C:\Users\jai_s\Desktop\2205Flask\2205Flask\venv\lib\site-packages\jinja2\environment.py", line 1301, in render
    self.environment.handle_exception()
  File "C:\Users\jai_s\Desktop\2205Flask\2205Flask\venv\lib\site-packages\jinja2\environment.py", line 936, in handle_exception
    raise rewrite_traceback_stack(source=source)
  File "C:\Users\jai_s\Desktop\2205Flask\2205Flask\app\templates\jobs.html", line 1, in top-level template code
    {% extends "base.html" %}
  File "C:\Users\jai_s\Desktop\2205Flask\2205Flask\app\templates\base.html", line 1, in top-level template code
    {% extends "bootstrap/base.html" %}
  File "C:\Users\jai_s\Desktop\2205Flask\2205Flask\venv\lib\site-packages\flask_bootstrap\templates\bootstrap\base.html", line 1, in top-level template code
    {% block doc -%}
  File "C:\Users\jai_s\Desktop\2205Flask\2205Flask\venv\lib\site-packages\flask_bootstrap\templates\bootstrap\base.html", line 4, in block 'doc'
    {%- block html %}
  File "C:\Users\jai_s\Desktop\2205Flask\2205Flask\venv\lib\site-packages\flask_bootstrap\templates\bootstrap\base.html", line 20, in block 'html'
    {% block body -%}
  File "C:\Users\jai_s\Desktop\2205Flask\2205Flask\venv\lib\site-packages\flask_bootstrap\templates\bootstrap\base.html", line 23, in block 'body'
    {% block content -%}
  File "C:\Users\jai_s\Desktop\2205Flask\2205Flask\app\templates\base.html", line 103, in block 'content'
    {% block app_content %}{% endblock %}
  File "C:\Users\jai_s\Desktop\2205Flask\2205Flask\app\templates\jobs.html", line 12, in block 'app_content'
    <p><a href="{{ url_for('main.create_jobs', id=job.id)}}">Edit Job</a></p>
  File "C:\Users\jai_s\Desktop\2205Flask\2205Flask\venv\lib\site-packages\jinja2\environment.py", line 485, in getattr
    return getattr(obj, attribute)
jinja2.exceptions.UndefinedError: 'job' is undefined
2023-08-10 13:14:31,992 INFO: Certi Tech startup [in C:\Users\jai_s\Desktop\2205Flask\2205Flask\app\__init__.py:71]
2023-08-10 13:16:59,176 INFO: Certi Tech startup [in C:\Users\jai_s\Desktop\2205Flask\2205Flask\app\__init__.py:71]
2023-08-10 14:28:59,030 INFO: Certi Tech startup [in C:\Users\Eliza Lenjani\Desktop\2205FlaskNew\app\__init__.py:71]
2023-08-14 07:11:24,840 INFO: Certi Tech startup [in C:\Users\Eliza Lenjani\Desktop\2205FlaskNew\app\__init__.py:71]
2023-08-14 07:38:08,410 INFO: Certi Tech startup [in C:\Users\Eliza Lenjani\Desktop\2205FlaskNew\app\__init__.py:71]
2023-08-14 07:56:47,078 INFO: Certi Tech startup [in C:\Users\Eliza Lenjani\Desktop\2205FlaskNew\app\__init__.py:71]
2023-08-14 08:00:34,219 INFO: Certi Tech startup [in C:\Users\Eliza Lenjani\Desktop\2205FlaskNew\app\__init__.py:71]
2023-08-14 08:05:51,872 INFO: Certi Tech startup [in C:\Users\Eliza Lenjani\Desktop\2205FlaskNew\app\__init__.py:71]
2023-08-14 08:09:27,991 INFO: Certi Tech startup [in C:\Users\Eliza Lenjani\Desktop\2205FlaskNew\app\__init__.py:71]
2023-08-14 08:10:00,211 INFO: Certi Tech startup [in C:\Users\Eliza Lenjani\Desktop\2205FlaskNew\app\__init__.py:71]
2023-08-14 08:12:46,433 INFO: Certi Tech startup [in C:\Users\Eliza Lenjani\Desktop\2205FlaskNew\app\__init__.py:71]
2023-08-14 09:38:09,042 INFO: Certi Tech startup [in C:\Users\Eliza Lenjani\Desktop\2205FlaskNew\app\__init__.py:71]
2023-08-14 09:42:57,826 INFO: Certi Tech startup [in C:\Users\Eliza Lenjani\Desktop\2205FlaskNew\app\__init__.py:71]
2023-08-14 09:50:31,895 INFO: Certi Tech startup [in C:\Users\Eliza Lenjani\Desktop\2205FlaskNew\app\__init__.py:71]
2023-08-14 09:50:40,339 ERROR: Exception on /aboutUs [GET] [in C:\Users\Eliza Lenjani\Desktop\2205FlaskNew\venv\lib\site-packages\flask\app.py:1414]
Traceback (most recent call last):
  File "C:\Users\Eliza Lenjani\Desktop\2205FlaskNew\venv\lib\site-packages\flask\app.py", line 2190, in wsgi_app
    response = self.full_dispatch_request()
  File "C:\Users\Eliza Lenjani\Desktop\2205FlaskNew\venv\lib\site-packages\flask\app.py", line 1486, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "C:\Users\Eliza Lenjani\Desktop\2205FlaskNew\venv\lib\site-packages\flask\app.py", line 1484, in full_dispatch_request
    rv = self.dispatch_request()
  File "C:\Users\Eliza Lenjani\Desktop\2205FlaskNew\venv\lib\site-packages\flask\app.py", line 1469, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**view_args)
  File "C:\Users\Eliza Lenjani\Desktop\2205FlaskNew\app\main\routes.py", line 20, in aboutUs
    return render_template('aboutUs.html',  title='About Us')
  File "C:\Users\Eliza Lenjani\Desktop\2205FlaskNew\venv\lib\site-packages\flask\templating.py", line 150, in render_template
    template = app.jinja_env.get_or_select_template(template_name_or_list)
  File "C:\Users\Eliza Lenjani\Desktop\2205FlaskNew\venv\lib\site-packages\jinja2\environment.py", line 1081, in get_or_select_template
    return self.get_template(template_name_or_list, parent, globals)
  File "C:\Users\Eliza Lenjani\Desktop\2205FlaskNew\venv\lib\site-packages\jinja2\environment.py", line 1010, in get_template
    return self._load_template(name, globals)
  File "C:\Users\Eliza Lenjani\Desktop\2205FlaskNew\venv\lib\site-packages\jinja2\environment.py", line 969, in _load_template
    template = self.loader.load(self, name, self.make_globals(globals))
  File "C:\Users\Eliza Lenjani\Desktop\2205FlaskNew\venv\lib\site-packages\jinja2\loaders.py", line 138, in load
    code = environment.compile(source, name, filename)
  File "C:\Users\Eliza Lenjani\Desktop\2205FlaskNew\venv\lib\site-packages\jinja2\environment.py", line 768, in compile
    self.handle_exception(source=source_hint)
  File "C:\Users\Eliza Lenjani\Desktop\2205FlaskNew\venv\lib\site-packages\jinja2\environment.py", line 936, in handle_exception
    raise rewrite_traceback_stack(source=source)
  File "C:\Users\Eliza Lenjani\Desktop\2205FlaskNew\app\templates\aboutUs.html", line 64, in template
    {% endblock %}
jinja2.exceptions.TemplateSyntaxError: Encountered unknown tag 'endblock'.
2023-08-14 09:50:43,252 ERROR: Exception on /aboutUs [GET] [in C:\Users\Eliza Lenjani\Desktop\2205FlaskNew\venv\lib\site-packages\flask\app.py:1414]
Traceback (most recent call last):
  File "C:\Users\Eliza Lenjani\Desktop\2205FlaskNew\venv\lib\site-packages\flask\app.py", line 2190, in wsgi_app
    response = self.full_dispatch_request()
  File "C:\Users\Eliza Lenjani\Desktop\2205FlaskNew\venv\lib\site-packages\flask\app.py", line 1486, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "C:\Users\Eliza Lenjani\Desktop\2205FlaskNew\venv\lib\site-packages\flask\app.py", line 1484, in full_dispatch_request
    rv = self.dispatch_request()
  File "C:\Users\Eliza Lenjani\Desktop\2205FlaskNew\venv\lib\site-packages\flask\app.py", line 1469, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**view_args)
  File "C:\Users\Eliza Lenjani\Desktop\2205FlaskNew\app\main\routes.py", line 20, in aboutUs
    return render_template('aboutUs.html',  title='About Us')
  File "C:\Users\Eliza Lenjani\Desktop\2205FlaskNew\venv\lib\site-packages\flask\templating.py", line 150, in render_template
    template = app.jinja_env.get_or_select_template(template_name_or_list)
  File "C:\Users\Eliza Lenjani\Desktop\2205FlaskNew\venv\lib\site-packages\jinja2\environment.py", line 1081, in get_or_select_template
    return self.get_template(template_name_or_list, parent, globals)
  File "C:\Users\Eliza Lenjani\Desktop\2205FlaskNew\venv\lib\site-packages\jinja2\environment.py", line 1010, in get_template
    return self._load_template(name, globals)
  File "C:\Users\Eliza Lenjani\Desktop\2205FlaskNew\venv\lib\site-packages\jinja2\environment.py", line 969, in _load_template
    template = self.loader.load(self, name, self.make_globals(globals))
  File "C:\Users\Eliza Lenjani\Desktop\2205FlaskNew\venv\lib\site-packages\jinja2\loaders.py", line 138, in load
    code = environment.compile(source, name, filename)
  File "C:\Users\Eliza Lenjani\Desktop\2205FlaskNew\venv\lib\site-packages\jinja2\environment.py", line 768, in compile
    self.handle_exception(source=source_hint)
  File "C:\Users\Eliza Lenjani\Desktop\2205FlaskNew\venv\lib\site-packages\jinja2\environment.py", line 936, in handle_exception
    raise rewrite_traceback_stack(source=source)
  File "C:\Users\Eliza Lenjani\Desktop\2205FlaskNew\app\templates\aboutUs.html", line 64, in template
    {% endblock %}
jinja2.exceptions.TemplateSyntaxError: Encountered unknown tag 'endblock'.
